core/domain/exp_jobs_one_off.py

```diff
@@ -810,7 +810,8 @@ def map(item):
             'cmd': 'change_role',
             'assignee_id': translator_id,
             'old_role': 'translator',
-            'new_role': 'voice artist'
+            # Using magic string because ROLE_TRANSLATOR is removed.
```

Comment:
This comment should be one line above (in general comments go above the line they are for).

Reply:
Done.